extensions/postgres/non_transactional.go

```diff
@@ -35,6 +36,14 @@ func (d dbSession) SelectCurrentProcessExecution(
 	return &row, err
 }
 
+const deleteCurrentProcessExecutionQuery = `DELETE
```

Comment:
For stopProcess API, we actually don't want to delete anything. The deletion will be done by the retention policy feature. Similar to Cadence/Temporal, all the history/execution records will be deleted after rention time beyond the execution is closed. 

However, this query is useful, you can keep it if you want

Reply:
True.